src/ops/info.rs

```diff
@@ -300,3 +340,20 @@ fn validate_locked_and_frozen_options(
     }
     Ok(())
 }
+
+fn try_get_msrv_from_nearest_manifest_or_ws(
+    nearest_manifest: &Option<PathBuf>,
+    ws: Option<&Workspace>,
+) -> Option<semver::Version> {
+    // Try to get the MSRV from the nearest manifest.
+    let rust_version = ws
+        .as_ref()
+        .and_then(|ws| {
+            nearest_manifest
+                .as_ref()
+                .and_then(|path| ws.members().find(|p| p.manifest_path() == path))
```

Comment:
I am not sure if this is a good way. Should I parse it directly? I guess searching it would be faster than re-prase it from disk?

Reply:
A `find` like this is the type of thing I would expect to see in the `cargo` code base

Reply:
Sound reasonable. Maybe we should support finding packages by pkgid and manifest_path.